tests/testUtils.js

```diff
@@ -84,5 +96,10 @@ module.exports = {
     .then(() => done())
     .catch((err) => done(err));
   }, // forceDeleteUser
-
+  // delete users
+  forceDeleteSubject(done) {
+    forceDelete(db.Subject, testStartTime)
+    .then(() => done())
+    .catch((err) => done(err));
+  }, // forceDeleteUser
 }; // exports
```

Comment:
testUtils.js has a generic method called "forceDelete", this takes in a model as its first argument. All you need to do is export the subject model doing something like u.db.Subject and pass the subject to forceDelete. This will take care of deleting the subject and you do not have to create another method just to delete the subjects.